schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts integer,
     neutered boolean,
     weight_kg numeric,
+    species text COLLATE pg_catalog."default",
     CONSTRAINT animals_pkey PRIMARY KEY (id)
```

Comment:
- [ ] According to the project requirements, I would suggest you add a new column called `species` with a string using the Alter SQL commands. Eg: `   ALTER TABLE animals ADD species varchar(255); `